core/src/main/java/org/testcontainers/containers/SelinuxContext.java

```diff
@@ -0,0 +1,16 @@
+package org.testcontainers.containers;
+
+import com.github.dockerjava.api.model.SELContext;
+
+/**
+ * Possible contexts for use with SELinux
+ */
+public enum SelinuxContext {
+    SHARED(SELContext.shared), SINGLE(SELContext.single), NONE(SELContext.none);
+
+    public final SELContext selContext;
```

Comment:
FYI since we use Lombok, you can remove "public final" and the constructor and just add a single `@Value` annotation to the class